h2gis-utilities/src/main/java/org/h2gis/utilities/dbtypes/DBTypes.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * H2GIS is a library that brings spatial support to the H2 Database Engine
+ * <http://www.h2database.com>. H2GIS is developed by CNRS
+ * <http://www.cnrs.fr/>.
+ *
+ * This code is part of the H2GIS project. H2GIS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation;
+ * version 3.0 of the License.
+ *
+ * H2GIS is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details <http://www.gnu.org/licenses/>.
+ *
+ *
+ * For more information, please consult: <http://www.h2gis.org/>
+ * or contact directly: info_at_h2gis.org
+ */
+package org.h2gis.utilities.dbtypes;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * Enumeration of database types.
+ *
+ * @author Erwan Bocher (CNRS 2021)
+ * @author Sylvain Palominos (UBS Chaire GEOTERA 2021)
+ */
+public enum DBTypes {
+    POSTGRESQL, POSTGIS, H2, H2GIS;
+
+
+    public Set<String> getReservedWords() {
```

Comment:
Doc
Reverved words will change with the db version ?

Reply:
Yes, but in a first time, only one version of the database will be supported, multi version support (like psql 9, 10, 11) will come in a later PR).